Doc/library/os.path.rst

```diff
@@ -488,6 +488,26 @@ the :mod:`glob` module.)
       Accepts a :term:`path-like object`.
 
 
+.. function:: splitroot(path)
+
+   Split the pathname *path* into a triad ``(drive, root, tail)`` where:
+
+   1. *drive* is an optional mount point, exactly like :func:`splitdrive`;
```

Comment:
I sympathise with the desire to keep repetition out of the docs, but I think it'll be friendlier to readers if we restate the description of _drive_ here, rather than forcing them to click through to the `splitdrive` function docs to find out when _drive_ will be an empty string or not.

Also, as a "typing guy" at heart -- "optional" has a special meaning for users of typing, where it means "could be `None`. I like the more explicit "Could be something meaningful, or the empty string" description in the `splitdrive()` docs.